### PR TITLE
Feature: Add outcome to claims in Member page

### PR DESCRIPTION
**Pull request by maxplt**
# Jira Issue: [IP-204](https://hedvig.atlassian.net/jira/software/projects/IP/boards/19?selectedIssue=IP-204)

## What?
Show outcome in claims list in the member page - claims tab

## Screenshots
image

## Optional checklist
- [ ] Updated `src/changelog.ts`
- [ ] Codescouted
- [ ] Unit tests written
- [ ] Tested locally

### CODE REVIEW

File: src/features/member/tabs/claims-tab/components/MemberClaimsList.tsx [hunk 1]
```diff
@@ -49,6 +49,10 @@ export const MemberClaimsList: React.FC<{ memberId: string }> = ({
 
   const claims = memberClaims ?? []
 
+  useEffect(() => {
+    console.log(claims)
+  }, [claims])
+
```
Comment (rasmusguterstam) [inline src/features/member/tabs/claims-tab/components/MemberClaimsList.tsx:55]: ❌

File: src/features/member/tabs/claims-tab/components/MemberClaimsList.tsx [hunk 2]
```diff
@@ -96,11 +100,20 @@ export const MemberClaimsList: React.FC<{ memberId: string }> = ({
                 </TableColumn>
 
                 <TableColumn>
-                  {claim.claimType ? (
-                    convertEnumToTitle(claim.claimType)
-                  ) : (
-                    <Placeholder>Not specified</Placeholder>
-                  )}
+                  <FlexVertically>
```
Comment (rasmusguterstam) [inline src/features/member/tabs/claims-tab/components/MemberClaimsList.tsx:103]: What's this? 😮 Why not just a `<Flex>`?

**Review by rasmusguterstam — APPROVED** [review]

File: src/features/member/tabs/claims-tab/components/MemberClaimsList.tsx (hunk 1 above)
Comment (maxplt) [reply src/features/member/tabs/claims-tab/components/MemberClaimsList.tsx:55]: Oops😰

File: src/features/member/tabs/claims-tab/components/MemberClaimsList.tsx (hunk 2 above)
Comment (maxplt) [reply src/features/member/tabs/claims-tab/components/MemberClaimsList.tsx:103]: I just get this one from Claims List Page

**Pull request merged**